src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const pricePerDay = 40;
+  const threeDayDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days < 3) {
+    total = pricePerDay * days;
```

Comment:
as you can see, `total` always has the initial value of `pricePerDay * days`, so you can put it there right away